README.md

```diff
@@ -115,22 +119,23 @@ For example:
 
 - To receive all updates of `Facebook` documents, the URL is `https://github.com/OpenTermsArchive/contrib-versions/commits/main/Facebook.atom`.
 - To receive all updates of the `Privacy Policy` from `Google`, the URL is `https://github.com/OpenTermsArchive/contrib-versions/commits/main/Google/Privacy%20Policy.md.atom`.
-## Reuse
 
-Open Terms Archive is built on several modules. Some of them are already exposed.
+## Importing as a module
 
-As `open-terms-archive` has not yet been published on npm, you can install it with: 
+Open Terms Archive exposes a JavaScript API to make some of its capabilities available in NodeJS. You can install it as an NPM module: 
 
 ```
 npm install ambanum/OpenTermsArchive#main
 ```
-### scripts
-For now, some scripts are exposed through the form of `bin` commands (You can find them in `./bin`)
 
-- `ota-lint-declarations` to lint a json declaration file
-- `ota-validate-declarations` to test if a json declaration file is valid.
+### CLI
 
-### features
+The following commands are available:
+
+- `ota-lint-declarations` to lint declarations. Use it with `./bin/ota-lint-declarations.sh`
+- `ota-validate-declarations` to test declarations. Use it with `./bin/ota-validate-declarations.sh`
```

Comment:
If this is made available through `npm install`, I don't think accessing shell scripts directly is the recommended way —or at least, we should mention `./node_modules/.bin` instead of `./bin`, since these paths are not directly accessible.

Reply:
I added a note on the fact that those commands are available globally